collector/requirements.txt

```diff
@@ -54,131 +55,11 @@ pandas==1.5.3
 parse==1.19.0
 pdfminer.six==20221105
 Pillow==9.5.0
-prompt-toolkit==1.0.14
+prompt-toolkit==3.0.38
 pycparser==2.21
 pydantic==1.10.8
 pyee==8.2.2
 Pygments==2.15.1
```

Comment:
I just realized some deps are missing that are required for the python langchain document loaders.
Can we add in `pypdf` & `docx2txt`. Not sure how those were not captured in the initial requirements.txt

```
docx2txt==0.8
pypdf==3.9.0
```

Reply:
yes, please, I ran into the pypdf thing as well...